examples/apps/collaborative-textarea/package.json

```diff
@@ -38,16 +38,16 @@
 		"webpack:dev": "webpack --env development"
 	},
 	"dependencies": {
-		"@fluid-example/example-utils": ">=2.0.0-internal.4.0.6 <2.0.0-internal.5.0.0",
-		"@fluid-experimental/react-inputs": ">=2.0.0-internal.4.0.6 <2.0.0-internal.5.0.0",
-		"@fluidframework/aqueduct": ">=2.0.0-internal.4.0.6 <2.0.0-internal.5.0.0",
+		"@fluid-example/example-utils": ">=2.0.0-internal.4.0.7 <2.0.0-internal.5.0.0",
```

Comment:
I noticed this is using the wider ranges which we no longer use (its ^ like instead of ~ like). Since you are updating all these ranges anyway, might be worth changing?

Reply:
Ok sure I didn't know if we wanted to backfill that to internal.4.0